### ISSUE TITLE
Update Kotlin runtime

### ISSUE THREAD

**Issue by milosmns** [Commit451/mailgun#2]
Latest Kotlin (1.2.21) has some very important fixes for CI/Server compilation tasks. I'm using 1.2.21 in my projects to address these issues.

I'm getting this from the `mailgun` library during builds: _(level: WARN)_

```
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.2.21/86407aed2dc197bd5f61d16e759c4ff43678732a/kotlin-stdlib-jdk8-1.2.21.jar (version 1.2)
    /var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-reflect/1.2.21/3159ff5936aa570a90050d385cb717fbb6c1723a/kotlin-reflect-1.2.21.jar (version 1.2)
    /var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.2.21/d64187eb2d9d1521be3421aa8c6774a8625cdde8/kotlin-stdlib-1.2.21.jar (version 1.2)
    /var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.2.21/88bfff5aa470143a83b0bc5ec00c0be8cabd7cad/kotlin-stdlib-jdk7-1.2.21.jar (version 1.2)
    /var/lib/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.1.4-2/272a21c30432c943d618008fbbd34762eb0d6c8a/kotlin-stdlib-jre7-1.1.4-2.jar (version 1.1)
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath or use '-Xskip-runtime-version-check' to suppress this warning
```

Can you pls update the runtime?

**Comment by milosmns** [comment]
Also RxJava is at 2.1.9 - https://mvnrepository.com/artifact/io.reactivex.rxjava2/rxjava

**Comment by Jawnnypoo** [comment]
Thanks for the reminder! These have been updated and released as 1.0.2. Enjoy!

**Comment by milosmns** [comment]
👍
